source/includes/fact-ssl-ssh-connection-warning.rst

```diff
@@ -0,0 +1,9 @@
+.. important::
+
+   Although |compass| supports connecting to a MongoDB instance through
+   an SSH tunnel via TLS/SSL, this configuration may lead to
+   unexpected behavior when connecting to a :manual:`secondary
+   </core/replica-set-secondary/>` member. After an election, the
+   secondary member to which you are connected may become the
+   :manual:`primary </core/replica-set-primary/>`, which may lead to
+   |compass-short| not properly displaying your data.
```

Comment:
Does it not properly display the data or is it that the data might not be as expected? Would the same scenario play out if you connected to a primary directly (i.e., could become secondary)?

Reply:
@imlucas, can you speak to this?

Reply:
See the tickets linked above in https://github.com/mongodb/docs-compass/pull/77/files#r218224451

Yes, the same issues primary vs. secondary. Really, its just that the connection errors in a non-friendly way.

Reply:
Oh and yes on the data is weird tip: you could be looking at stale data, but the odd error case is much more likely